### PR TITLE
Feature#34 create model and crud check

### PR DESCRIPTION
**Pull request by Reiyirak**
# Description

- What?: We made the backend for the check-in and check-out that the employees will use. The view we made shows the clock and the input fields for the employee number and password, along with two radio buttons making it possible for the employee for him to check-in and check-out for the day.
- Why?: These changes complete one of the issues that we established at the start of the sprint and also let us keep moving forward on the backend for our project.

## Screenshots

image
image
image
image

### CODE REVIEW

**Review by M-Cordova-M — APPROVED** [review]
LGTM

**Review by juperameza — APPROVED** [review]
LGTM

File: app/helpers/checks_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ChecksHelper
+end
```
Comment (YilKanda) [inline app/helpers/checks_helper.rb:2]: If you don't occupy it, you can delete it.

File: test/controllers/checks_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class ChecksControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (YilKanda) [inline test/controllers/checks_controller_test.rb:6]: Do not leave code in comments.

File: test/models/check_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CheckTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (YilKanda) [inline test/models/check_test.rb:6]: Do not leave code in comments.

**Review by YilKanda — DISMISSED** [review]

File: app/helpers/checks_helper.rb (hunk 1 above)
Comment (Yaxkin121) [reply app/helpers/checks_helper.rb:2]: It's actually necessary, thx!

**Review by dagp-ucol — APPROVED** [review]

**Pull request merged**